### DISCUSSION TITLE
Filter access=customers from parking search?

### DISCUSSION THREAD

**Discussion by thompehw** [osmandapp/OsmAnd#11454] [Q&A] [answered]
Is it possible to display car parking on the map without showing parking tagged with access=customers? This would show where I could leave the car without being naughty.

I know how to search and display parking. And if I define a custom search for Transportation>Parking then I have found the filters option which seems to offer everything except what I'm looking for!

Thanks for any tips.

**Comment by vshcherb** [comment, marked as answer]
That would make sense.

**Reply by thompehw** [reply]
Yes, it's odd but the basic use case of "Where can I leave my car?" needs more than the detailed  filters currently offered.

As well as filtering-out access=customers, filtering access=private. from the search would help a lot.
